Sources/MongoSwift/MongoCollection.swift

```diff
@@ -48,6 +63,9 @@ public struct MongoCollection<T: Codable> {
         self.namespace.collection! // swiftlint:disable:this force_unwrapping
     }
 
+    /// The options used to initialize this `MongoCollection`.
+    public let options: MongoCollectionOptions?
```

Comment:
can we make this `private`? Otherwise I worry the duplication with the other fields (e.g. `readConcern` and `writeConcern`) may confuse users.